app/src/main/kotlin/com/bq/arch_example/ui/pokemonlist/PokemonListActivity.kt

```diff
@@ -47,10 +47,10 @@ class PokemonListActivity : AppCompatActivity(), KodeinAware {
 
         pokemonListViewModel.setup()
 
-        pokemonListViewModel.getPokemonListLiveData().observe(this) { viewData ->
-            toggleViewsVisibility(viewData.pokemonListRes, list_content, list_loading, error, View.GONE)
+        pokemonListViewModel.getPokemonListLiveData().observe(this) { resource ->
```

Comment:
Maybe the extension for the `observeResource` should be added to the template

Reply:
70e5a3c